versioned_docs/version-latest/community/index.md

```diff
@@ -0,0 +1,83 @@
+---
+id: awesome
+title: Awesome Subspace
+sidebar_position: 1
+description: Awesome Community List for Subspace Labs and Network
+keywords:
+    - Awesome
+    - Community
+    - Subspace
+---
+
+:::caution
+
+Please note, all community provided resources are **non-official**. For clarification please refer to [official materials](/docs/intro).
```

Comment:
Similarly, this link should be `(../intro)`

Reply:
Resolved in commit 97dd5cf & 9a76b88